data/campaigns/Under_the_Burning_Suns/scenarios/04_Descending_into_Darkness.cfg

```diff
@@ -575,9 +575,9 @@
 
         [message]
             speaker=Scylla
-            message= _ "Yesss, we remember the oath...  Sssslay them all!" # wmllint: no spellcheck
-# leaving this here because it might have been a reference to the alien?  It really isn't clear.
-#            message= _ "Sssslay them all! In the name of the Ssscaled One!" # wmllint: no spellcheck
+            message= _ "Yesss, we remember the oath... Sssslay them all!" # wmllint: no spellcheck
+            # leaving this here because it might have been a reference to the alien?  It really isn't clear.
+            #            message= _ "Sssslay them all! In the name of the Ssscaled One!" # wmllint: no spellcheck
```

Comment:
The whitespace _after_ this `#` character should be replaced with a single space. @Elvish-Hunter I wonder, is this automatable in wmllint?

Reply:
I generally prefer ifdef-ing out code over commenting out code, but in the meantime, I've fixed the whitespace in ad68ef0